assets/goal-bot/GoalBotArchitecture.iuml

```diff
@@ -0,0 +1,65 @@
+'
+' PlantUML diagram for the Goal Bot architecture.
+' Please re-generate the image (.png) after updating the code
```

Comment:
Forgive me if I've missed something, but how do we generate the image from the iuml?

Reply:
@harryBedfordSL - Good question. I'll document that in the README. I'm using Visual Code to do this for me.

Reply:
Looks like you can do it with a command

java -jar plantuml.jar ".../assets/goal-bot"

but [requires Java 8 or onwards](https://plantuml.com/command-line)

Reply:
@Ch0live  / @harryBedfordSL - thanks for feedback. I've recommending using the IDE extension. Being a python project, not every developer with have Java or the plantuml JAR file.